packages/benchmarks/lit-element/stub1/tachometer.json

```diff
@@ -0,0 +1,63 @@
+{
+  "$schema": "https://raw.githubusercontent.com/Polymer/tachometer/master/config.schema.json",
+  "sampleSize": 30,
+  "timeout": 0,
+  "benchmarks": [
+    {
+      "url": "lit-element/stub1/stub1.html",
+      "browser": {
+        "name": "chrome",
+        "headless": true
+      },
+      "measurement": {
+        "mode": "performance",
+        "entryName": "render"
+      },
+      "expand": [
+        {
+          "name": "local"
```

Comment:
For names in the table, maybe something like this would be clearer?
* ~local~ → This change
* ~lit-next~ → Tip-of-tree
* ~npm~ → Previous release

?

Reply:
Done.